src/main/scala/fundamentals/level04/LogParser.scala

```diff
@@ -45,7 +45,8 @@ object LogParser {
   trait LogMessage
  
  /**
-   * Once you have defined your data types, remove `import Types._` from [LogParserTest.scala](src/test/scala/fundamentals/level04/LogParserTest.scala).
+   * - Once you have defined your data types, remove `import Types._` from [LogParserTest.scala](src/test/scala/fundamentals/level04/LogParserTest.scala).
+   * - Comment out the entire contents of src/test/scala/fundamentals/level04/LogParserTest.scala
```

Comment:
should this be:

> Comment out the entire contents of src/test/scala/fundamentals/level04/Types.scala

Reply:
Yes!

Maybe do: 

```
Once you have defined your data types:
1. Remove...
2. Comment out...
```

I think numbering the actions makes it extra clear.

Thanks Stili.

Reply:
🤦‍♂️